test/web3c.js

```diff
@@ -143,3 +176,18 @@ describe('Web3', () => {
     // won't always be able to decode the returned log.
   }).timeout(timeout);
 });
+
+/**
+ * @returns a web3c client with a mocked signer for response signature validation.
+ */
+function web3cMockSigner(gw) {
+  let mockSigner = {
+    verify: (_sign, _key, _timestamp) => {  }
+  };
+  let _web3c = new web3c(gw);
```

Comment:
why an `_` here?

Reply:
Because the tests currently define `web3c`, so we can't use that as the name.

What we should do instead is rename all uses of `web3c` to `Web3c` and then use `web3c` here, but that's outside the scope of these changes. Alternatively we could do something like `client`, but I don't like that since it's different from the convention used in most other places I've seen.

Reply:
the initial underscore was also confusing because it normally indicates private but that isn't the case in this case because it's the value being returned.

i'd prefer client or instance or a similar name.

Reply:
But It is exactly private to this file and test. `client` and `instance` are unclear imo and can refer to either the web3.js object or to the contract depending on the contract. Again, the proper fix is not to use client or instance, it's to change `web3c` to `Web3c`.